app/models/user.rb

```diff
@@ -1,7 +1,7 @@
 class User < ApplicationRecord
-  has_many :comments, foreign_key: 'author_id', dependent: :destroy
-  has_many :posts, foreign_key: 'author_id', dependent: :destroy
-  has_many :likes, foreign_key: 'author_id', dependent: :destroy
+  has_many :comments, foreign_key: 'author_id'
+  has_many :posts, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
 
   validates :Name, presence: true
   after_initialize :set_defaults
```

Comment:
- [ ] Good job so far 👏 This line above is not allowing me to add the seed data. Therefore, It was hard for me to run the app. Please remove it so that  the next reviewer will be able to test it 😃 👍

Reply:
Corrected accordingly.